com.woltlab.wcf/option.xml

```diff
@@ -1022,7 +1022,15 @@ XING</selectoptions>
 			<option name="module_url_unfurling">
 				<categoryname>message.general.unfurl</categoryname>
 				<optiontype>boolean</optiontype>
+				<enableoptions>url_unfurling_load_images</enableoptions>
 				<defaultvalue>1</defaultvalue>
+				<showorder>1</showorder>
+			</option>
+			<option name="url_unfurling_no_images">
```

Comment:
I would prefer to see the logic of the option reversed. In other words, an option that activates the loading of images rather than deactivating it.